customers.go

```diff
@@ -181,6 +195,17 @@ func (api API) UpdateCustomer(customer *Customer, customerUUID string) (*Custome
 		result)
 }
 
+// UpdateCustomerV2 updates one customer in API.
+//
+// See https://dev.chartmogul.com/v1.0/reference#update-a-customer
+func (api API) UpdateCustomerV2(customer *UpdateCustomer, customerUUID string) (*UpdateCustomer, error) {
+	result := &UpdateCustomer{}
```

Comment:
While the `*UpdateCustomer` serves well the input, the response is not well represented by it.
Use the `*Customer` struct.
For details, see all the additional fields that are there: https://dev.chartmogul.com/reference#update-a-customer